src/qofonoradiosettings.h

```diff
@@ -28,7 +28,8 @@ class QOFONOSHARED_EXPORT QOfonoRadioSettings : public QOfonoModemInterface
 {
     Q_OBJECT
     Q_PROPERTY(QString modemPath READ modemPath WRITE setModemPath NOTIFY modemPathChanged)
-    Q_PROPERTY(QString technologyPreference READ technologyPreference WRITE setTechnologyPreference NOTIFY technologyPreferenceChanged)
+    Q_PROPERTY(QString availableTechnologies READ availableTechnologies NOTIFY availableTechnologiesChanged)
+    Q_PROPERTY(QStringList technologyPreference READ technologyPreference WRITE setTechnologyPreference NOTIFY technologyPreferenceChanged)
```

Comment:
I think you have these two backwards. technologyPreference is a string and availableTechnologies should be a stringlist.